src/NzbDrone.Core/Movies/MoveMovieService.cs

```diff
@@ -42,6 +42,12 @@ private void MoveSingleMovie(Movie movie, string sourcePath, string destinationP
                 return;
             }
 
+            if (sourcePath.Equals(destinationPath))
+            {
+                _logger.ProgressInfo("'{0}' is already in the right location '{1}'.", movie.Title, destinationPath);
```

Comment:
Let's reword this a bit and use the default `ToString()` for the movie object which will come through in logs like this `[2012 (2009)][tt1190080, 14161]`

```suggestion
                _logger.ProgressInfo("{0} is already in the specified location '{1}'.", movie, destinationPath);
```